src/convertToObject.js

```diff
@@ -1,18 +1,27 @@
 'use strict';
 
 /**
- * Implement convertToObject function:
- *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
- * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
+  const string = sourceString.split('\n')
+    .filter(item => item.length > 0)
+    .map(item => item.split(':'))
+    .map(word => word.map(restyle => restyle.replace(';', '').trim()))
+    .filter(item => item.length > 1);
+
+  for (let i = 0; i < string.length; i++) {
+    const key = string[i][0];
+    const value = string[i][1];
```

Comment:
What are `key` and `value`? Would be great to specify what exactly these values are